### DISCUSSION TITLE
After Running Jailbreak Composer will not open

### DISCUSSION THREAD

**Discussion by RunKilted** [garrynewman/Control4.Jailbreak#29] [Q&A]
After running the jailbreak script and certs Composer Pro will no longer open. I have uninstalled Composer, and went as far as restoring the PC to a previous restore point. Still after installing Composer Pro it will not open. I previously had composer running as I was a C4 programmer for 10+ years professionally (not anymore). I checked composers logs but it's not building any. Any ideas? How do i completely remove the script and all changes?

**Comment by skinner6695** [comment]
I am also seeing this issue. New laptop built with Windows 10 22H2. Composer Pro 3.1.1 newly installed. Didnt open Composer. Then ran the Jailbreak app, then Composer will never open, it doesnt even show in Task Manager. I tried to replaced the config file and no change. Event viewer doesnt provide any events either.

**Reply by skinner6695** [reply]
I found I had to switch to 3.3.1 for it to launch, but then failed to connect to Director.

**Comment by freshcodemonger** [comment]
I found composer 3.3.1 worked.  I tried earlier versions but they wouldn't launch.  3.3.1 connected just fine to my 3.2.1 system.  It did have some warning about not connecting to services online or something but worked just fine to pull my settings down.

**Comment by RunKilted** [comment]
Thank you, i'll give that a try.
